packages/react-aria-components/src/Table.tsx

```diff
@@ -1178,12 +1181,14 @@ export interface CellRenderProps {
    * Whether the cell is currently hovered with a mouse.
    * @selector [data-hovered]
    */
-  isHovered: boolean
+  isHovered: boolean,
+  /**
+   * The unique id of the cell.
+   **/
+  id?: Key
 }
 
 export interface CellProps extends RenderProps<CellRenderProps> {
-  /** The unique id of the cell. */
-  id?: Key,
```

Comment:
Sorry to add a comment to a closed PR, but I'm just dealing with upgrading RAC in our codebase, and this removal of `Cell`'s `id` prop is causing some type errors, and also head scratching for me! I'm curious about why it was removed, as we've been using that prop for a while now, and occasionally referencing `Cell`'s by their `id`, and so maybe have been using the component incorrectly. Would appreciate any insight to help with the upgrade!

Reply:
That looks like a mistake. Thanks for reporting. It looks like it was moved to the block above instead of being copied.
```
export interface CellProps extends RenderProps<CellRenderProps>
```
looks a lot like
```
export interface CellProps extendsCellRenderProps
```
if you don't open the definition of RenderProps

apologies for the inconvenience, I've opened a PR to fix it. [fix: Add Table Cell id prop back into the Cell types](https://github.com/adobe/react-spectrum/pull/7983#top)